### DISCUSSION TITLE
stoicism/evening_ritual/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#19] [General]
# stoicism/evening_ritual/

https://shanenull.com/stoicism/evening_ritual/

**Comment by shane0** [comment]
# bullet journals

- butts did not allow embed on https://youtu.be/xiCtt3VLy9E
- Satipaṭṭhāna  https://www.dhammatalks.org/suttas/MN/MN10.html

**Reply by shane0** [reply]
https://github.com/shane0/buddhism/issues/127

**Reply by shane0** [reply]
- see form for notes
- [ ] add journal for diamond sutra
- heart sutra includes 6 dusts + 5 skhandas
- 6 dusts: https://shanenull.com/buddhism/2021/docs_2020/videos/suttas/#sutta-148
- 5 skhandas video added
- diamond sutra cuts through all arbitrary conceptions
- heart sutra no wisdom phenomenon + neumonon no attainment

**Reply by shane0** [reply]
- [ ] write books for my sons + family before i die

**Reply by shane0** [reply]
- https://shanenull.com/buddhism/2023/confusion/#unanswerable-questions

**Comment by shane0** [comment]
## the meditative mind

|               | Meditative Mind | Untrained Mind |
| ------------- | --------------- | -------------- |
| Attention     | Focused         | Distracted     |
| Awareness     | Expanded        | Limited        |
| Thoughts      | Observing       | Identifying    |
| Emotions      | Regulating      | Reactive       |
| Perception    | Clear           | Clouded        |
| Judgment      | Non-judgmental  | Judgmental     |
| Response      | Intentional     | Automatic      |
| Concentration | Developed       | Scattered      |
| Insight       | Cultivated      | Absent         |
| Clarity       | Increased       | Decreased      |
| Equanimity    | Present         | Absent         |

**Comment by shane0** [comment]
88. How To Write a Chinese Poem

A well-known Japanese poet was asked how to compose a Chinese poem.

"The usual Chinese poem is four lines," he explained. "The first line contains the initial phrase; the second line, the continuation of that phrase; the third line turns from this subject and begins a new one; and the fourth line brings the first three lines together. A popular Japanese song illustrates this:

Two daughters of a silk merchant live in Kyoto.
The elder is twenty, the younger, eighteen.
A soldier may kill with his sword,
But these girls slay men with their eyes."
